sdk-consumer/tsconfig.json

```diff
@@ -0,0 +1,111 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
+
+    /* Projects */
+    // "incremental": true,                              /* Save .tsbuildinfo files to allow for incremental compilation of projects. */
```

Comment:
Should we just get rid of all the commented-out stuff?

Reply:
Yeah will do, this is generated by `tsc init`